docs/host_limit.rst

```diff
@@ -17,3 +17,26 @@ Event data example:
 The value for "hosts" can be a comma delimited string of multiple host names, but typically
 it is a single host. This is useful to restrict a remedy playbook to run only on the problematical host
 that emits a monitored event. Be sure that the source plugin already sends this metadata.
+
+
+Inserting hosts to meta
+-----------------------
+
+The plugins may not send metadata like for example the `ansible.eda.webhook` plugin which
+sends the arbitrary payload under the `payload` key.
+
+To accommodate this, the EDA collection provides the `insert_hosts_to_meta` filter,
+allowing any plugin to customize the value of `event.meta.hosts` based on the contents
+of a specific key in the event.
+
+Example:
+
+.. code-block:: yaml
```

Comment:
@Alex-Izquierdo Can we also add this into the webhook source plugin notes in the collection

Reply:
Sure we can. But what we need is definitely address the whole documentation for source plugins.

Reply:
are you fine to merge this?